galvan-support/pom.xml

```diff
@@ -57,6 +57,12 @@ limitations under the License.
       <groupId>org.terracotta</groupId>
       <artifactId>galvan</artifactId>
       <version>${galvan.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.terracotta</groupId>
+          <artifactId>terracotta-utilities-port-chooser</artifactId>
```

Comment:
Is this temporary until Galvan gets a new release?

Reply:
No, it is not.  This module has a direct dependency on port-chooser and Maven's busted dependency convergence enforcement pulls in the base version declared by galvan instead of the version actually used or resolved in this module.  Icky but necessary.